assets/sites/default/settings.acquia.php

```diff
@@ -53,23 +53,28 @@
     );
   }
 
-  // {{{1 Conditionally manage memory.
+  // Conditionally manage memory.
+  $high_memory_paths = array();
+  // Things that break in ODFE.
   if (isset($conf['default']['odfe']) && $conf['default']['odfe']['enabled']) {
-    $high_memory_paths = array(
+    $high_memory_paths += array(
       'admin',
       'node/add',
       'node/%node/edit',
       'node/%node/moderation',
       'file/ajax',
       'api/action/datastore/search.json',
     );
+  }
+  
+  // ODSM edit forms.
+  $high_memory_paths[] = 'admin/config/services/odsm/edit';
 
-    // Standarize node edit paths for validation.
-    $current_path = preg_replace("/\d+/", '%node', $_GET['q']);
-    foreach ($high_memory_paths as $high_memory_path) {
-      if ((strpos($current_path, $high_memory_path) === 0)) {
-        ini_set('memory_limit', '512M');
-      }
+  // Standarize node edit paths for validation.
+  $current_path = preg_replace("/\/\d+/", '/%node', $_GET['q']);
```

Comment:
Under the original regex, `data_json_1_1` was being "standardized" to `data_json_%node_%node`